tools/devops/automation/templates/build/configure.yml

```
@@ -147,16 +147,16 @@ steps:
   continueOnError: true
   workingDirectory: $(Build.SourcesDirectory)\tools\devops 
 
-- ${{ if eq(variables['Build.Reason'], 'Schedule') }}:
-  - task: OneLocBuild@2
-    condition: and(succeeded(), eq(variables.isMain, 'True'))
-    continueOnError: true
-    env:
-      SYSTEM_ACCESSTOKEN: $(System.AccessToken)
-    inputs:
-      locProj: '$(Build.SourcesDirectory)\Localize\LocProject.json'
-      outDir: '$(Build.ArtifactStagingDirectory)'
-      isCreatePrSelected: true
+- task: OneLocBuild@2
+  condition: and(succeeded(), eq(variables.isMain, 'True'))
+  continueOnError: true
+  env:
+    SYSTEM_ACCESSTOKEN: $(System.AccessToken)
+  inputs:
+    locProj: '$(Build.SourcesDirectory)\Localize\LocProject.json'
+    outDir: '$(Build.ArtifactStagingDirectory)'
+    isCreatePrSelected: eq(variables['Build.Reason'], 'Schedule')
+    ${{ if eq(variables['Build.Reason'], 'Schedule') }}:
       isAutoCompletePrSelected: false
       prSourceBranchPrefix: 'locfiles'
       repoType: gitHub
```

This means that the defaults are used when we are not on a 'Schedule' builds, can you make this more explicit for those of us that do not know them?
```suggestion
    ${{ if eq(variables['Build.Reason'], 'Schedule') }}:
      isAutoCompletePrSelected: false
      prSourceBranchPrefix: 'locfiles'
      repoType: gitHub
    ${{ if eq(variables['Build.Reason'], 'Schedule') }}:
       ## other settings go here.
```

---

Good point! I looked at the sample and when we do not have isCreatePrSelected = true, we only need to define packageSourceAuth and patVariable with regards to github.

```
steps:
- task: cesve.one-loc-build.one-loc-build.OneLocBuild@2
  displayName: 'Localization Build: Localize/LocProject.json'
  inputs:
    packageSourceAuth: patAuth
    patVariable: '$(OneLocBuildPat) ```

---

There are always default values:

* locProj:  "Localize/LocProject.json"
* name: "outDir: "$(Build.ArtifactStagingDirectory)"
* lclSource: lclFilesInRepo"
* lclPackageId: ""
* lclPackageVersion: "highestLclPackageVersion"
* lclPackageVersionVariable: : ""
* isEnablePseudoLanguagesSelected: "false",
* isCreatePrSelected: "false",
* repoType: "azureReposGit",
* prSourceBranchPrefix:  "locfiles",
* azureReposGitPatVariable:: "",
* gitHubPatVariable:  "",
* isAutoCompletePrSelected: "true",
* isDeletePrSourceBranchSelected:  "true",
* isUseLfLineEndingsSelected: "false",
* gitHubPrMergeMethod:"merge",
* packageSourceAuth: "credentialProviderAuth",
* patVariable: "",
* lsBuildPackageId: "lsBuildXLoc",
* lsBuildXLocPackageVersion: "",
* lsBuildLegacyXLocPackageVersion:  "",
* lsBuildWarningLevel: "0",
* xLocConsoleLoggingLevel: "information",
* nugetCliVersion: "",
  
Looking at it, we are interested in 

**isCreatePrSelected** : 'Create a pull request to check-in localized files to the repo"', defaults to false, so should be false except when we are in 'Schedule' (correct in the PR
**isAutoCompletePrSelected**: "Automatically complete a pull request for localized file check-in skipping review." DEFAULT TRUE MUST BE FALSE This is where you are wrong since you added under the Schedule if it means that is true, should always be false no matter what.

Those are the two I have checked, please review all of them with the docs/src I fwd you.

---

Thanks, I looked at this and will just assign the other inputs without the conditional since I think it makes more sense to have those values instead of the defaults! The only one that **needs** to be different is that  isCreatePrSelected I'll keep that one with the conditional!